packages/opentelemetry-plugin-http/package.json

```diff
@@ -39,19 +39,29 @@
     "access": "public"
   },
   "devDependencies": {
+    "@opentelemetry/scope-async-hooks": "^0.0.1",
     "@types/mocha": "^5.2.7",
     "@types/node": "^12.6.9",
+    "@types/semver": "^6.0.1",
+    "@types/shimmer": "^1.0.1",
+    "@types/sinon":"^7.0.13",
+    "sinon": "^7.4.0",
     "c8": "^5.0.1",
     "codecov": "^3.5.0",
     "gts": "^1.1.0",
     "mocha": "^6.2.0",
+    "nock": "^10.0.6",
     "ts-mocha": "^6.0.0",
     "ts-node": "^8.3.0",
     "typescript": "^3.5.3"
   },
   "dependencies": {
     "@opentelemetry/core": "^0.0.1",
     "@opentelemetry/node-tracer": "^0.0.1",
-    "@opentelemetry/types": "^0.0.1"
+    "@opentelemetry/types": "^0.0.1",
+    "@types/nock": "^10.0.3",
+    "nock": "^10.0.6",
```

Comment:
both `nock` and `@types/nock` should be dev dependencies right ?

Reply:
Right, it was also duplicate in devDeps...